pxtlib/docsrender.ts

```diff
@@ -230,6 +230,8 @@ namespace pxt.docs {
             return injectHtml(templ, mparams, ["ITEMS"])
         }
 
+        params["selectLang"] = lf("Select Language");
```

Comment:
do you really need these changes? Any modification of docsrender.ts require a cloud update.

Reply:
Today I'm going to focus on trying to remove any of my changes that need cloud updates and replace them. I hope I can make this work in all the PR I have open. :)